tests/unit/core/agents/agent_components/test_streaming_debug.py

```
@@ -0,0 +1,476 @@
+        assert _truncate_prefix(text) == text
+
+    def test_long_text_truncated(self):
+        text = "x" * (DEBUG_STREAM_PREFIX_MAX_LEN + 50)
+        result = _truncate_prefix(text)
+        assert len(result) == DEBUG_STREAM_PREFIX_MAX_LEN
+
+
+class TestCapturePreFirstDeltaText:
+    def test_first_delta_seen_returns_existing(self):
+        event = SimpleNamespace(part=SimpleNamespace(content="new"))
+        result = _capture_pre_first_delta_text(event, True, "existing")
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)